### ISSUE TITLE
Implement the MPRIS OpenUri command

### ISSUE THREAD

**Issue by mrsobakin** [jeffvli/sonixd#317]
**Is your feature request related to a problem? Please describe.**
The user is unable to change playing song via cli.

**Describe the solution you'd like**
Implement the MPRIS `OpenUri` command.

**Describe alternatives you've considered**
N/A

**Additional context**
Given that `OpenUri` requires... well, Uri, several schemes could be implemented.
For example:
1. Opening song by its `subsonic` or `jellyfin` id
`by-id://120843393563b2b0c1bcc627578612b6`
This will be the easiest scheme to implement, and I consider it as an essential minimum
2. Opening song by its name
`by-name://Queen%20-%20Bohemian%20Rhapsody`
Basically searching by name and selecting best match
3. etc.

And it's not limited just by songs. Schemes to support playing the playlists or the artists could also be implemented.
